src/modules/m_cloaking.cpp

```diff
@@ -42,6 +42,7 @@ static const char base32[] = "0123456789abcdefghijklmnopqrstuv";
 class CloakUser : public ModeHandler
 {
  public:
+	bool active;
```

Comment:
Can you initialise this to false in the constructor as non-local primitives do not have a guaranteed default value. Also, while you are altering the constructor can you alter it to be in the correct style, i.e.

```
ModuleCloaking()
	: active(false)
	, cu(this)
	// rest of initialisers here in the same style
{
}
````

Reply:
(not related to this change)

It might be nice to pick up initialization of members in the headers instead. That would require being C++11-only, but that ought to be acceptable by now, anyway.